tests/conftest.py

```diff
@@ -121,9 +121,9 @@ def mq_test_df(mq_file_name):
     return pgio.ms_file_reader(mq_file_name)
 
 
-@pytest.fixture
-def ftrs_file_name():
-    return str(DATA_DIR / "ftrs_test_data.ftrs")
+@pytest.fixture(params=["ftrs_test_data_5.2.ftrs", "ftrs_test_data_3.11.ftrs"])
+def ftrs_file_name(request):
```

Comment:
The `request` parameter is one of those magical pytest "keywords"... Makes me sad, but this seems to the "proper" way to do this sort of thing.

Reply:
I've not seen fixtures parameterised in this manner before, thanks for the example. :+1: 

What alternative would you prefer to have seen though?